### DISCUSSION TITLE
Draft release on PR to main "No Changes"

### DISCUSSION THREAD

**Discussion by cmiller-jetty** [release-drafter/release-drafter#1364] [Q&A]
What I want to do..

- on PR open to main
  - draft a release
- on PR approval / merge into main
  - publish release

Currently, 

- on PR open to main and label added ✅ 
- draft a release ❌ 
  - drafts a release, while the PR is labeled, correctly -- but
    - defaults to a patch release (e.g. X.X.X+1)
    - shows "No Changes"
![Screenshot 2023-10-17 at 1 30 21 PM](https://github.com/release-drafter/release-drafter/assets/101212479/bef50315-3fd9-4f08-b9fd-df26fca1b879)

- on PR approval / merge into main
  - publish release ✅ 
  - Once this is published (using the same action, in a different workflow -  but setting `with:  publish: true`)
    - it uses the appropriate semver, e.g. (minor) 
    - it updates the changelog appropriately. 
![Screenshot 2023-10-17 at 1 31 32 PM](https://github.com/release-drafter/release-drafter/assets/101212479/eab281ea-426b-4209-a51c-22cdbfeb46ba)

Is there a way to get the draft release to show the proper semver, changelog, etc.. while being triggered from a PR label being added?
```on:
  pull_request:
    types:
        - labeled
```

**Comment by cmiller-jetty** [comment]
[jetersen](https://github.com/jetersen) any quick thoughts here would be super helpful, even if its "not possible"

**Reply by jetersen** [reply]
The metadata is only available once the PR is merged.

If you're talking about PRs that are already merged but then labeled afterwards. 
You should be able to add the trigger for PR labeled and run the release-drafter.

If that's not working, I am not sure.
